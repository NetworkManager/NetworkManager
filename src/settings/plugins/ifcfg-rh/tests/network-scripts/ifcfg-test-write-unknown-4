# test what happens with multiple defines of a name.
#
# Note that svGetValue() will return "l4x", which
# isn't correct in terms of shell-parsing. But we
# don't consider only line-by-line, thus this is
# expected.
#
# Also note that setting NAME will replace the last
# occurrence, and delete all previous once.

#L1
NAME=l2

#L2
NAME=l3

some_key1=''
some_key2=$'\U0x'
some_key3=$'x\U0'

#L4
NAME='
NAME=l4x
'

#Lx-1
NAME2=not-visible
#Lx-2
NAME2='invalid
#Lx-3

#Ly-1
NAME3='invalid
#Ly-2
NAME3=name3-value
#Ly-3
