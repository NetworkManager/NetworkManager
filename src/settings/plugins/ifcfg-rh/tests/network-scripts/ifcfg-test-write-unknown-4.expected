# test what happens with multiple defines of a name.
#
# Note that svGetValue() will return "l4x", which
# isn't correct in terms of shell-parsing. But we
# don't consider only line-by-line, thus this is
# expected.
#
# Also note that setting NAME will replace the last
# occurrence, and delete all previous once.

#L1

#L2

some_key1=''
some_key2=$'\U0x'
some_key3=$'x\U0'

#L4
NAME=set-by-test1
#NM: '

#Lx-1
#Lx-2
NAME2=set-by-test2
#Lx-3

#Ly-1
#Ly-2
NAME3=set-by-test3
#Ly-3
